File: app/src/main/java/hu/krisz768/bettertuke/BetterTukeApplication.java
package hu.krisz768.bettertuke;

import android.app.Application;

import com.google.android.material.color.DynamicColors;

public class BetterTukeApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        DynamicColors.applyToActivitiesIfAvailable(this);
    }
}

File: app/src/main/java/hu/krisz768/bettertuke/BetterTukeApplication.kt
package hu.krisz768.bettertuke

import android.app.Application
import com.google.android.material.color.DynamicColors

class BetterTukeApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        DynamicColors.applyToActivitiesIfAvailable(this) //Hello World
    }
}